lib/datadog/tracing/contrib/sinatra/env.rb

```diff
@@ -40,24 +38,15 @@ def header_to_rack_header(name)
             "HTTP_#{name.to_s.upcase.gsub(/[-\s]/, '_')}"
           end
 
-          # Was a Sinatra already traced in this request?
-          # We don't want to create spans for intermediate Sinatra
-          # middlewares that don't match the request at hand.
-          def middleware_traced?(env)
```

Comment:
Not used.

Reply:
What was this introduced by? I don't think I authored it, but I imagine it had to be added with purpose. Is that purpose obsolete? Double check this isn't AppSec related... cc @lloeki

Reply:
I don't seem to recall this being AppSec related